ios/RNSScreen.h

```diff
@@ -114,6 +118,9 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)notifyTransitionProgress:(double)progress closing:(BOOL)closing goingForward:(BOOL)goingForward;
 - (void)notifyDismissCancelledWithDismissCount:(int)dismissCount;
 - (BOOL)isModal;
+- (BOOL)isPresentedAsNativeModal;
+- (BOOL)isFullscreenModal;
+- (BOOL)isTransparentModal;
```

Comment:
If these methods are used in `RNSScreen.mm` only and I believe they are -- we do not need to declare them in `@interface` -- these can be declared & defined entirely in `RNSScreen.mm`.

Reply:
Good point! I'll remove `isFullscreenModal` and `isTransparentModal`, as `isPresentedAsNativeModal` is being used in RNSScreenStack 😁

Reply:
Introduced in https://github.com/software-mansion/react-native-screens/pull/1802/commits/ae03a777dff550b45fe0a31055e154fef7ea49ca.